docs/modules/ROOT/pages/vshn-managed/redis/index.adoc

```diff
@@ -0,0 +1,12 @@
+= Managed by VSHN - Redis
```

Comment:
Can we call it "Redis by VSHN"? (also for PostgreSQL then)

Reply:
Fixed. For Postgresql see: #30